Assets/Tests/Common/EOSTestBase.cs

```diff
@@ -77,20 +95,23 @@ public WaitUntilDone(float timeout, Func<bool> predicate)
         private bool _initialized;
         private bool _successfulLogin;
 
+        protected EOSManager EOSManager { get; set; }
+
         /// <summary>
         /// Initialize the EOSManager once before attempting to log in.
         /// </summary>
         [OneTimeSetUp]
         public void SetupScene()
         {
             eosObject = new GameObject();
-            var eosManager = eosObject.AddComponent<EOSManager>();
-            EOSManager.Instance.Init(eosManager);
+            EOSManager = eosObject.AddComponent<EOSManager>();
+            EOSManager.Instance.Init(EOSManager);
```

Comment:
This should not be inside the base class `EOSTestBase` because there are unit tests that extend from `EOSTestBase` that do not require the presence of an `EOSManager` instance, and requiring them to nonetheless make an instance is incorrect.

Reply:
This was already happening, though. This is the [OneTimeSetUp]. All that is changed here is I'm assigning a variable to a place it can be found.